src/Spectre.libHeatmapDataScaling.Tests/Main.cpp

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Instead of duplicating the content of the entry point for tests, which is common for all test projects, you may as well just include the already existing file `src/Common/Main.cpp`.